src/components/atoms/Input/Input.tsx

```diff
@@ -0,0 +1,36 @@
+import {TextInput, ImageBackground} from 'react-native';
+import {styles} from './styles';
+import {colors} from '../../../static/consts/colors';
+
+type TInputProps = {
+  placeholder: string;
+  value: string;
+  onChange: (e: string | React.ChangeEvent<any>) => void;
+  onBlur: (e: any) => void;
```

Comment:
Same as before, teraz dajesz jakieś any i inne chuje, a jak rozszerzysz to o typ z react-native, to masz to od buta

Reply:
done